server/php-engine.cpp

```diff
@@ -2089,7 +2093,7 @@ int main_args_handler(int i, const char *long_option) {
         host = "127.0.0.1";
       }
 
-      StatsHouseClient::init(host, port);
+      StatsHouseMetrics::init_client(host, port);
```

Comment:
I think it is not good to divide initialization into several methods, because potentially we can get a non-valid state

Reply:
after the above changes here is single initialization, renamed to just `init`